ui/lib/apps/KeyViz/components/KeyViz.tsx

```diff
@@ -220,16 +220,18 @@ const KeyViz = (props) => {
       />
     )
   ) : (
-    <div className="keyviz_disabled_container">
-      <h2>{t('keyviz.settings.disabled_desc_title')}</h2>
-      <div className="keyviz_disabled_desc">
-        <p>{t('keyviz.settings.disabled_desc_line_1')}</p>
-        <p>{t('keyviz.settings.disabled_desc_line_2')}</p>
-      </div>
-      <Button type="primary" onClick={openSettings}>
-        {t('keyviz.settings.open_setting')}
-      </Button>
-    </div>
+    <Result
+      title={t('keyviz.settings.disabled_desc_title')}
+      subTitle={
+        t('keyviz.settings.disabled_desc_line_1') +
```

Comment:
Seems that we can merge them together, to be something like:

```yaml
keyviz:
  settings:
    disabled_result:
      title: xxx
      subTitle: xxx
```

In this way it matches the `<Result>` better.

Reply:
Good point! will update it soon!